backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/SpecificationEntity.kt

```diff
@@ -1,44 +1,25 @@
 package com.tngtech.apicenter.backend.connector.database.entity
 
-import com.tngtech.apicenter.backend.domain.entity.Specification
-import com.tngtech.apicenter.backend.domain.entity.Version
 import org.hibernate.search.annotations.Field
 import org.hibernate.search.annotations.Indexed
+import org.hibernate.search.annotations.IndexedEmbedded
 import java.util.UUID
+import javax.persistence.CascadeType
 import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.GeneratedValue
 import javax.persistence.Id
+import javax.persistence.OneToMany
 
 @Entity
 @Indexed
 data class SpecificationEntity(
     @Id @GeneratedValue val id: UUID,
     @Field val title: String,
     @Field @Column(columnDefinition = "TEXT") val description: String?,
-    @Field val version: String,
-    @Field @Column(columnDefinition = "TEXT") val content: String,
+    @IndexedEmbedded @OneToMany(
+        mappedBy = "specification",
+        cascade = [CascadeType.ALL]
+    ) val versions: List<VersionEntity>,
```

Comment:
Do we know that they're sorted?

Reply:
They are implicitly sorted by creation date (which is exactly what we want).

Reply:
I will make the sorting explicit now.